src/styles/global/_lecture-evaluation-transition.scss

```
@@ -0,0 +1,50 @@
+}
+
+@keyframes lecture-evaluation-wrapper-burst {
+  to {
+    opacity: 0;
+    transform: scale(1.025);
+    filter: brightness(1.18) saturate(0.92);
+  }
+}
+
+@keyframes lecture-evaluation-card-reveal {
+  from {
```

Stylelint (17.13.0)

[error] Expected "from" to be "0%" (keyframe-selector-notation)

(keyframe-selector-notation)